kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/EventProvider.java

```diff
@@ -13,12 +14,22 @@
 public interface EventProvider {
 
   /**
-   * List all events in an organization
+   * List events in an organization
    * 
    * @param organizationId organization id
+   * @param startBefore return only events starting before the date
+   * @param startAfter return only events starting after the date
+   * @param endBefore return only events ending before the date
+   * @param endAfter return only events ending after the date
+   * @param order order
+   * @param orderDirection ordering direction
+   * @param firstResult first index of results
+   * @param maxResults maximum number of results
    * @return events organization events
    */
-  public List<Event> listOrganizationEvents(OrganizationId organizationId);
+  public List<Event> listOrganizationEvents(OrganizationId organizationId, OffsetDateTime startBefore, OffsetDateTime startAfter, 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)